test/JSON-Schema-Test-Suite/tests/draft_2020-12/definitions.json

```diff
@@ -1,12 +1,12 @@
 [
     {
         "description": "validate definition against metaschema",
-        "schema": {"$ref": "http://json-schema.org/draft-07/schema#"},
+        "schema": {"$schema": "http://json-schema.org/draft/2020-12/schema"},
         "tests": [
             {
                 "description": "valid definition schema",
                 "data": {
-                    "definitions": {
+                    "$defs": {
```

Comment:
I don't think this is correct; the test is still failing (er, passing when it should fail). Could someone help me understand the structure here better?

Reply:
I have barely stuck my toe into json schema definitions, so it is entirely possible that your question is mostly over my head. But draft 2020-12 allows either `definitions` or `$defs` as a keyword. Is that what you were expecting to fail? Because if so, it should, in fact, pass. My PR #338 allows either to pass precisely for this reason.

Reply:
I think I wrote that comment when I was still trying to get my bearings on stuff.
I think you'll agree that `$defs` is "correct", which `definitions` is the backward compatible safety net, no?
At least, according to https://json-schema.org/draft/2020-12/schema

Reply:
>I think you'll agree that $defs is "correct", which definitions is the backward compatible safety net, no?

That's the way I read it as well.